tasks/install.yml

```diff
@@ -62,6 +62,8 @@
   retries: "{{ devture_playbook_help_container_retries_count }}"
   delay: "{{ devture_playbook_help_container_retries_delay }}"
   until: result is not failed
+  changed_when: result.rc == 0
+  failed_when: result.rc != 0
```

Comment:
`failed_when` is OK, but maybe that's the default behavior anyway?

-----

`changed_when` probably won't work. I manually did a `docker pull` for an image I don't have and the exit code appears to be `0`. So, `docker pull` returns `0` every time the command completes successfully (regardless of whether the image is there or not), so we cannot use this "0 = success" indicator for determining if "the command performed a change or not".

The way `changed_when` is now, it will always report "no changes" (even if it did actually pull an image), which is misleading. The current behavior (not having `changed_when`) probably defaults to always reporting that there's a change, which is similarly misleading (in the other direction). The default behavior is probably better.